package.json

```diff
@@ -71,8 +53,9 @@
     "ciwin:all": "npm ci && npm run install:components",
     "csslint": "stylelint **/*.css",
     "csslintfix": "stylelint **/*.css --fix",
-    "install:all": "npm install && npm run install:components",
+    "install:all": "npm install && npm run install:markbind && npm run install:components",
     "install:components": "cd frontend/components && npm install",
+    "install:markbind": "npm install src/lib/markbind",
```

Comment:
hence, for example if you only updated the markbind core package's dependencies, simply run `install:markbind`.
If multiple packages had their dependencies  upgraded, run `install:all` instead. (documented under **workflow** under dev guide)